src/shared/views/topMenu/TopMenu/topMenu.css

```diff
@@ -0,0 +1,7 @@
+.topMenu {
+  background-color: #55ACEF;
+  height: 34px;
+  padding: 5px 5px;
```

Comment:
I think this can be removed and the height can just be made 50px. No padding. we are auto centering so there doesnt seem to be a need for padding. Maybe left-right padding if you think.